### PR TITLE
feat: Add support for ignoring standalone tasks and role tasks in the graph view

### PR DESCRIPTION
**Pull request by Eltryo**
## Premise
When running the ansible-playbook-grapher on a huge project folder the resulting svg or mermaid graph happens to be quiet confusing with all the task nodes being rendered. So far there was no option to view all roles (also nested roles) without viewing the corresponding tasks, so I implemented a feature to do exactly that.

## Changes
- Add entry for the new option ```--only-roles``` in README.md
- Add the option in ```cli.py``` and adjust ```grapher.py``` and ```parser.py``` accordingly
- Update docstrings accordingly
- Add logic to prevent tasks from being added
- Lint project with the corresponding make target
- Add cli and parser tests

## Screenshots
without ```--only-roles```
group-roles-by-name
with ```--only-roles```
group-roles-by-name-only-roles

### CODE REVIEW

**Comment by haidaraM** [comment]
Hi, thanks for your contribution. I just approved the workflows, and I'm rebasing your branch to see if the checks pass.

**Comment by haidaraM** [comment]
Hmm, I have to make a few changes to the workflows so that they work with forks. Will keep you in touch.

**Comment by Eltryo** [comment]
> Hmm, I have to make a few changes to the workflows so that they work with forks. Will keep you in touch.

Ah alright, thank you!

File: ansibleplaybookgrapher/cli.py [hunk 1]
```diff
@@ -167,6 +168,13 @@ def _add_my_options(self) -> None:
         """
         self.parser.prog = __prog__
 
+        self.parser.add_argument(
+            "--only-roles",
+            dest="only_roles",
+            action="store_true",
+            help="Ignore all tasks when rendering graphs.",
```
Comment (haidaraM) [inline ansibleplaybookgrapher/cli.py:175]:
```suggestion
            help="Only display the roles in the graph (ignoring the tasks)",
```

**Review by haidaraM — COMMENTED** [review]
Small changes to make. 

Other than that, this seems to be a reasonable feature to add

**Review by haidaraM — APPROVED** [review]

**Pull request merged**